modules/AlphaWalletFoundation/AlphaWalletFoundation/Transactions/EtherscanSingleChainTransactionProvider.swift

```diff
@@ -30,6 +30,8 @@ class EtherscanSingleChainTransactionProvider: SingleChainTransactionProvider {
             ercTokenDetector: ercTokenDetector)
     }()
 
+    public private (set) var state: TransactionProviderState = .pending
```

Comment:
Is `.pending` only used as the initial state?

Reply:
yes, right